cdflow.py

```diff
@@ -116,15 +116,24 @@ def _get_component_name_from_git_remote():
     return name
 
 
-def get_platform_config_path(argv):
+def _get_platform_config_path_arg(iterator):
     try:
-        flag_index = argv.index('--platform-config')
-    except ValueError:
+        return next(iterator)
+    except StopIteration:
         raise MissingPlatformConfigError()
-    try:
-        return argv[flag_index+1]
-    except IndexError:
+
+
+def get_platform_config_paths(argv):
+    paths = []
+    iterator = iter(argv)
+    for arg in iterator:
+        if arg == '--platform-config':
```

Comment:
Is this condition so we handle two invocation cases, like:
- `cdflow --platform-config /path/to/platform/config`
- `cdflow --platform-config=/path/to/platform/config`

?

Reply:
Yes, it at least handles passing those through to the argument handling of cdflow-commands. I'm not 100% sure whether this handles it but i think it should (I think accepting either is the usual convention - right?).

Reply:
I think it's a good practice - definitely have seen CLI tools that don't handle both :-)